load.go

```diff
@@ -54,7 +55,7 @@ func loadAction(clicontext *cli.Context) error {
 }
 
 func loadImage(in io.Reader, clicontext *cli.Context) error {
-	client, ctx, cancel, err := newClient(clicontext)
+	client, ctx, cancel, err := newClient(clicontext, containerd.WithDefaultPlatform(platforms.DefaultStrict()))
```

Comment:
client.Import should receive the platform option, not client itself

Reply:
@AkihiroSuda  the `platformMatcher` refers to  client option
https://github.com/containerd/containerd/blob/96df20cf770479393342047a62182961f9205d62/import.go#L128